dev-db/redis/redis-7.0.1.ebuild

```diff
@@ -55,9 +55,6 @@ PATCHES=(
 src_prepare() {
 	default
 
-	# unstable on jemalloc
```

Comment:
We've lost this, is that ok?

Reply:
Yes, it is ok. Sorry, I was too tired and lazy to write some summary of those commits yesterday, but it is actually explained in commit messages. This issue seems to affect `6.0` only, see https://github.com/gentoo/gentoo/pull/25831/commits/fefe8b0398f983501e754ddd1444811d2ac315fb. It was fixed in `6.2*` and it does not affect `7.0*` either, at least those tests work for me, see https://github.com/gentoo/gentoo/pull/25831/commits/b325b5d9f167f9ed34de76b87afa48ec73ae3000.

Reply:
It's my bad for not reading the commit messages! Great work here.